### PR TITLE
Model to support Green Building Registry

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
Added functionality by green building registry.

#### What's this PR do?
Combines 5 PRs into one. 

#### How should this be manually tested?
Unit tests, monkey test.

#### What are the relevant tickets?
#1378 #1379 #1381 #1382 #1389 

#### Screenshots (if appropriate)
n/A

#### Definition of Done:
- [x] Is there appropriate test coverage? (e.g. ChefSpec, Mocha/Chai, Python, etc.)
- [x] Does this PR require a Selenium test? (e.g. Browser-specific bugs or complicated UI bugs)
- [x] Does this PR require a regression test? All fixes require a regression test.
- [x] Does this add new dependencies? If so, does PIP, npm, bower requirements need to be updated?

### CODE REVIEW

**Comment by nllong** [comment]
@fablet  -- looks like all the tests are now passing! Thanks for working on this. I imagine that this will be merged down to develop shortly.

File: docs/source/spelling_wordlist.txt [hunk 1]
```diff
@@ -241,6 +248,8 @@ util
 utils
 uwsgi
 uWSGI
+validator
```
Comment (Myoldmopar) [inline docs/source/spelling_wordlist.txt]: Surprising this needs to be whitelisted.

File: seed/lib/mappings/test_mapping_data.py [hunk 2]
```diff
@@ -29,11 +29,11 @@ def test_mapping_data_init(self):
 
         # verify that the data loaded as expected
         fake_data_0 = {
-            "js_type": "",
+            "js_type": u"",
             "name": "address_line_1",
             "schema": "BEDES",
             "table": "PropertyState",
-            "type": "CharField",
+            "type": u"CharField",
```
Comment (Myoldmopar) [inline seed/lib/mappings/test_mapping_data.py]: Do we need to coerce u'' on any other string literals in this object?

File: seed/renderers.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+class SEEDJSONRenderer(JSONRenderer):
+    """
+    Custom JSON Renderer.
+
+    returns results in the format {'status': 'success', 'data': data} or
+        {'status': 'error', 'message': error message}
+
+    if self.data_name is set on the view its will be used in place of data
```
Comment (Myoldmopar) [inline seed/renderers.py]: Super minor typo `its`

File: seed/tests/test_certification_models.py [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    def test_gap_properties(self):
+        """Test properties on GreenAssessmentProperty."""
+        self.assertEqual('Green Test Score', self.gap.name)
+        self.assertEqual('Green TS Inc', self.gap.body)
+        self.assertEqual('SCR', self.gap.recognition_type)
+        self.assertEqual('Score', self.gap.recognition_description)
+        self.assertEqual(self.start_date.year, self.gap.year)
+        self.assertEqual(
+            self.green_assessment.organization, self.gap.organization
+        )
```
Comment (Myoldmopar) [inline seed/tests/test_certification_models.py]: Lots of asserts here :+1:

Comment (nllong) [inline seed/tests/test_certification_models.py]: so good

File: docs/source/spelling_wordlist.txt (hunk 1 above)
Comment (nllong) [inline docs/source/spelling_wordlist.txt]: Same here. I spelled it outloud multiple times to make sure that I wasn't going crazy.

File: seed/lib/mappings/test_mapping_data.py (hunk 2 above)
Comment (nllong) [inline seed/lib/mappings/test_mapping_data.py]: no we don't need to coerce it. We flip/flop on this syntax and I don't think it really matters at the moment.

File: seed/renderers.py (hunk 3 above)
Comment (nllong) [inline seed/renderers.py]: converted to it's.

**Pull request merged**